### PR TITLE
Update Auth API auth0.js samples to v8

### PR DESCRIPTION
**Pull request by angelaLazar**
First part of updating the Auth API auth0.js samples to v8 

https://trello.com/c/pN2CGhqi/826-authentication-api-update-auth0-js-sam
ples-to-v8

### CODE REVIEW

**Comment by angelaLazar** [comment]
@mpaktiti I think this should be ready to merge now

File: articles/api/authentication/_login.md [hunk 1]
```diff
@@ -229,59 +195,41 @@ GET https://${account.namespace}/authorize?
 ```
 
 ```javascript
-<script src="${auth0js_url}"></script>
+// Script uses auth0.js v8. See Remarks for details.
```
Comment (mpaktiti) [inline articles/api/authentication/_login.md:198]: @jeffreylees This auth0.js script is not valid, it has username/password while the passive authentication is just a redirect to the Auth0 Login Page that will show the Login Widget where the user can login manually with email and password. 

I think we need something like the [buildAuthorizeUrl](https://github.com/auth0/auth0.js#auth0authentication). Can you please check with brucke and find the correct script for this case? Maybe we should add this to the auth0.js index as well.

Comment (jeffreylees) [reply articles/api/authentication/_login.md:198]: Yes, `buildAuthorizeUrl` is what we need to use. It takes the params that an authorize request would need.

```
webAuth.client.buildAuthorizeUrl({
  clientID: 'YOUR_CLIENT_ID', // string
  responseType: 'token', // code or token
  redirectUri: 'http://example.com',
});
```

Something like this? Also will want to recommend setting a `state` probably.

Comment (mpaktiti) [reply articles/api/authentication/_login.md:198]: Thanks Jeff! I tried it, works fine. 👍

**Pull request merged**